### PR TITLE
Catkinizing meta-package cob_perception_data

### PR DESCRIPTION
**Pull request by ipa-jsf**
compiling on my PC and catkin_lind passed without warnings

@ipa-fxm Please review

### CODE REVIEW

File: cob_tesseract_data/CMakeLists.txt [hunk 1]
```diff
@@ -1,30 +1,4 @@
...
-#common commands for building c++ executables and libraries
-#rosbuild_add_library(${PROJECT_NAME} src/example.cpp)
-#target_link_libraries(${PROJECT_NAME} another_library)
-#rosbuild_add_boost_directories()
-#rosbuild_link_boost(${PROJECT_NAME} thread)
-#rosbuild_add_executable(example examples/example.cpp)
-#target_link_libraries(example ${PROJECT_NAME})
+project(cob_tesseract_data)
+find_package(catkin REQUIRED)
+catkin_package()
```
Comment (fmessmer) [inline cob_tesseract_data/CMakeLists.txt:4]: there are know install tags in this CMakeLists.txt

File: cob_read_text_data/CMakeLists.txt [hunk 2]
```diff
@@ -1,30 +1,4 @@
...
-#common commands for building c++ executables and libraries
-#rosbuild_add_library(${PROJECT_NAME} src/example.cpp)
-#target_link_libraries(${PROJECT_NAME} another_library)
-#rosbuild_add_boost_directories()
-#rosbuild_link_boost(${PROJECT_NAME} thread)
-#rosbuild_add_executable(example examples/example.cpp)
-#target_link_libraries(example ${PROJECT_NAME})
+project(cob_read_text_data)
+find_package(catkin REQUIRED)
+catkin_package()
```
Comment (fmessmer) [inline cob_read_text_data/CMakeLists.txt]: there are know install tags in this CMakeLists.txt

**Comment by fmessmer** [comment]
Please add install tags:
- for the directories with the data
- for the python scripts
  see http://wiki.ros.org/catkin/CMakeLists.txt#Optional_Step:_Specifying_Installable_Targets

Also, I assume this file was unintentionally added to the repo:
https://github.com/ipa-jsf/cob_perception_data/blob/hydro_dev/cob_read_text_data/dictionary/full-dictionary_eng%20%28original%29

**Comment by fmessmer** [comment]
It compiles in Hydro and Indigo!

**Comment by fmessmer** [comment]
@ipa-jsf could you also please set the default branch of this repository to hydro_dev?

**Comment by ipa-jsf** [comment]
@ipa-fxm Fixed install tags and file name

**Comment by ipa-jsf** [comment]
@ipa-fxm Added missing files

**Pull request merged**